exercises/algorithms/README.md

```diff
@@ -12,72 +12,55 @@ For this assignment, we've provided the starter project above.
 
 ## :pushpin: Sorting
 
-### Merge Sort<sup>1</sup>
```

Comment:
Why is Merge Sort being removed?

Reply:
For the same reason that linear search is being removed — there's just too much for them to do in a single session. We're cutting the exercises from 4 to 2

Reply:
Cool, have the slides been updated to reflect this?

Reply:
Yep, we'd already skipped these slides for the current cohort